File: src/01_Fibonacci_numbers/fibonacci_numbers.cpp
#include <iostream>

using namespace std;

int main(void) {
    int n = 0;
    cin >> n;
    long int temp = 0;
    long int a = 0;
    long int b = 1;
    for (int i = 1; i < n; i++) {
        temp = a + b;
        a = b;
        b = temp;
    }
    cout << b << endl;
    return 0;
}

File: src/01_Fibonacci_numbers/fibonacci_numbers.go
package main

import "fmt"

func main() {
	var n int
	var temp uint64
	var a uint64 = 0
	var b uint64 = 1
	fmt.Scan(&n)
	for i := 1; i < n; i++ {
		temp = a + b
		a = b
		b = temp
	}
	fmt.Println(b)
}